app/assets/javascripts/components/overview/my_articles.jsx

```diff
@@ -51,6 +51,7 @@ const MyArticles = React.createClass({
               student={this.props.current_user}
               assignments={assigned}
               prefix={I18n.t('users.my_assigned')}
+              tooltip_message="Assign yourself an article to edit"
```

Comment:
These tooltip messages should go through I18n.

`An assigned article is one that you plan to create or improve.`